docs/design/INTEGRATE-HEALTHCHECK-SERVICE

```diff
@@ -0,0 +1,56 @@
+# Integrating EdFi.AdminConsole.HealthCheckService into EdFi.Ods.AdminApi with Quartz.NET
+
+## Overview
+
+This document describes the design and process for integrating the `EdFi.AdminConsole.HealthCheckService` into the `EdFi.Ods.AdminApi` application, leveraging Quartz.NET for scheduled and on-demand execution of health checks.
+
+---
+
+## Goals
+
+* Enable scheduled health checks of ODS API instances via Quartz.NET.
+* Allow on-demand triggering of health checks via an API endpoint.
+* Ensure only one health check job runs at a time to prevent data conflicts.
+* Centralize health check logic in `EdFi.AdminConsole.HealthCheckService`.
+
+---
+
+## Architecture
+
+### Components
+
+* **HealthCheckService**: Core service that performs health checks across tenants and instances.
```

Comment:
At this point, its just a "feature" of the Admin API, right? Rather than a stand-alone service?